src/styles/main.scss

```diff
@@ -0,0 +1,129 @@
+$bodyfont: Roboto, sans-serif;
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0 auto;
+  display: flex;
+  padding: 0;
+  gap: 57px;
```

Comment:
Using 'gap' on the 'body' element is not a common practice and may not work as expected across all browsers, especially since 'body' is not typically used as a flex container for direct children. Consider applying 'gap' within specific flex containers instead.